src/functions-and-arrays.js

```diff
@@ -29,9 +149,47 @@ const wordsUnique = [
   'bring'
 ];
 
+
+const uniquifyArray = (array) => {
+  if (array.length === 0) {
+    return null;
+  }
+
+  let uniqueArray = [];
+
+  for (let i = 0; i < array.length; i++){
```

Comment:
Maybe instead of using the for loop you could use the forEach one, it will go around the test Jasmine is not passing:

array.forEach(function (word) {
    if (!uniqueArray.includes(word)) {
      uniqueArray.push(word)
    };
  })